WordPress/src/test/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterViewModelTest.kt

```diff
@@ -1,22 +1,51 @@
 package org.wordpress.android.ui.activitylog.list.filter
 
 import kotlinx.coroutines.InternalCoroutinesApi
+import org.assertj.core.api.Assertions.assertThat
 import org.junit.Before
 import org.junit.Test
 import org.wordpress.android.BaseUnitTest
 import org.wordpress.android.TEST_DISPATCHER
+import org.wordpress.android.test
+import org.wordpress.android.ui.activitylog.list.filter.ActivityLogTypeFilterViewModel.ListItemUiState
+import org.wordpress.android.ui.activitylog.list.filter.ActivityLogTypeFilterViewModel.UiState
+import org.wordpress.android.ui.activitylog.list.filter.ActivityLogTypeFilterViewModel.UiState.Content
 
 @InternalCoroutinesApi
 class ActivityLogTypeFilterViewModelTest : BaseUnitTest() {
     private lateinit var viewModel: ActivityLogTypeFilterViewModel
 
     @Before
     fun setUp() {
-        viewModel = ActivityLogTypeFilterViewModel(TEST_DISPATCHER)
+        viewModel = ActivityLogTypeFilterViewModel(TEST_DISPATCHER, TEST_DISPATCHER)
     }
 
     @Test
-    fun `skeleton test`() {
-        // Skeleton test.
+    fun `fullscreen loading shown, when screen initialized`() = test {
+        val uiStates = initObservers().uiStates
+
+        viewModel.start()
+
+        assertThat(uiStates[0]).isInstanceOf(UiState.FullscreenLoading::class.java)
+    }
+
+    @Test
+    fun `section header gets added as first item in the list`() {
+        val uiStates = initObservers().uiStates
```

Comment:
Minor (🔍 ): The `uiStates` val is unused here. Maybe just using `initObservers()` is enough for this test as a setup. Wdyt?

Reply:
Fixed in 416893abd3da953154503e839c77a482efbec525